### PR TITLE
implement ABBAA picking method

### PR DESCRIPTION
**Pull request by thboss**

### CODE REVIEW

File: bot/cogs/match.py [hunk 1]
```diff
@@ -283,6 +300,24 @@ def all_ready(reaction, user):
             else:
                 raise ValueError(f'Team method "{team_method}" isn\'t valid')
 
+            voice_channel_one = self.bot.get_channel(697938536850128978)
```
Comment (B3none) [inline bot/cogs/match.py:303]: These look like they're specific to your server.

File: bot/cogs/match.py [hunk 2]
```diff
@@ -283,6 +300,24 @@ def all_ready(reaction, user):
...
+            await voice_channel_one.edit(name=f'Team {team_one[0].display_name}')
+            voice_channel_two = self.bot.get_channel(697938656752828416)
```
Comment (B3none) [inline bot/cogs/match.py:305]: These look like they're specific to your server.

File: bot/cogs/match.py [hunk 3]
```diff
@@ -302,11 +337,18 @@ def all_ready(reaction, user):
...
             return True  # Everyone readied up
 
+
     @commands.command(usage='teams {captains|autobalance|random}',
                       brief='Set or view the team creation method (Must have admin perms)')
     @commands.has_permissions(administrator=True)
     async def teams(self, ctx, method=None):
         """ Set or display the method by which teams are created. """
+        if ctx.message.channel.id != 705391032891867166:
+            text_channel = ctx.bot.get_channel(705391032891867166)
```
Comment (B3none) [inline bot/cogs/match.py:347]: These look like they're specific to your server.

File: bot/cogs/match.py [hunk 4]
```diff
@@ -332,6 +374,12 @@ async def teams(self, ctx, method=None):
     @commands.has_permissions(administrator=True)
     async def captains(self, ctx, method=None):
         """ Set or display the method by which captains are selected. """
+        if ctx.message.channel.id != 705391032891867166:
+            text_channel = ctx.bot.get_channel(705391032891867166)
```
Comment (B3none) [inline bot/cogs/match.py:378]: These look like they're specific to your server.

File: bot/cogs/queue.py [hunk 5]
```diff
@@ -43,6 +43,12 @@ async def update_last_msg(self, ctx, embed):
     @commands.max_concurrency(1, per=commands.BucketType.guild, wait=True)  # Only process one command per guild at once
     async def join(self, ctx):
         """ Check if the member can be added to the guild queue and add them if so. """
+        if ctx.message.channel.id != 705391032891867166:
+            text_channel = ctx.bot.get_channel(705391032891867166)
```
Comment (B3none) [inline bot/cogs/queue.py:47]: These look like they're specific to your server.

File: bot/cogs/queue.py [hunk 6]
```diff
@@ -95,6 +101,12 @@ async def join(self, ctx):
     @commands.command(brief='Leave the queue (or the bursted queue)')
     async def leave(self, ctx):
         """ Check if the member can be remobed from the guild and remove them if so. """
+        if ctx.message.channel.id != 705391032891867166:
+            text_channel = ctx.bot.get_channel(705391032891867166)
```
Comment (B3none) [inline bot/cogs/queue.py:105]: These look like they're specific to your server.

File: bot/cogs/queue.py [hunk 7]
```diff
@@ -111,6 +123,12 @@ async def leave(self, ctx):
     @commands.command(brief='Display who is currently in the queue')
     async def view(self, ctx):
         """ Display the queue as an embed list of mentioned names. """
+        if ctx.message.channel.id != 705391032891867166:
+            text_channel = ctx.bot.get_channel(705391032891867166)
```
Comment (B3none) [inline bot/cogs/queue.py:127]: These look like they're specific to your server.

File: bot/cogs/queue.py [hunk 8]
```diff
@@ -145,6 +169,12 @@ async def remove(self, ctx):
     @commands.has_permissions(kick_members=True)
     async def empty(self, ctx):
         """ Reset the guild queue list to empty. """
+        if ctx.message.channel.id != 705391032891867166:
+            text_channel = ctx.bot.get_channel(705391032891867166)
```
Comment (B3none) [inline bot/cogs/queue.py:173]: These look like they're specific to your server.

File: bot/cogs/queue.py [hunk 9]
```diff
@@ -166,6 +196,12 @@ async def remove_error(self, ctx, error):
     @commands.has_permissions(administrator=True)
     async def cap(self, ctx, *args):
         """ Set the queue capacity. """
+        if ctx.message.channel.id != 705391032891867166:
+            text_channel = ctx.bot.get_channel(705391032891867166)
```
Comment (B3none) [inline bot/cogs/queue.py:200]: These look like they're specific to your server.

File: bot/cogs/match.py (hunk 2 above)
Comment (thboss) [reply bot/cogs/match.py:305]: @b3none yes I just testing

**Comment by cameronshinn** [comment]
I implemented a quick solution to have an enforced team draft order on [this branch](https://github.com/csgo-league/csgo-league-bot/tree/team-draft-enhancements). It was blocking Scotty from making improvements to the team draft embed. If you want to keep working on how to make the bot automatically move people we should start some discussion on how we want that to work.

**Pull request closed without merging**